layouts/partials/default.html

```diff
@@ -40,9 +40,16 @@
           <!-- end "Edit This Page" button -->
 
         <div class="p-lg-5 p-3 bg-white">
-          <h2 class="mb-5" id="title">{{ .Title }}</h2>
+          <h2 id="title">{{ .Title }}</h2>
+          {{if .Params.author}}
+          <span class="text-dark">{{ .Params.author }}</span>
+          {{if .Params.authorEmail}}
+           <span>-</span>
+          <a href = "mailto:{{ .Params.authorEmail }}">{{.Params.authorEmail}}</a>
```

Comment:
The HTML attribute here does not validate; it should be written without whitespace as `href="mailto:{{…`

Reply:
Resolved in commit 8441a60013588a7fc7b7d103f869b401057914b2